From fff0f259563980c02b9833f6c34563840edbec7c Mon Sep 17 00:00:00 2001
From: Ding Wei <leo.ding@rock-chips.com>
Date: Thu, 29 Oct 2020 11:05:14 +0800
Subject: [PATCH] video: rockchip: mpp: rename for read only procfs node

using "-" instead of "_", which means the node is read only.

Change-Id: If66764daca51f96bdccd5324e251fc9bd956691d
Signed-off-by: Ding Wei <leo.ding@rock-chips.com>
---
 drivers/video/rockchip/mpp/mpp_rkvenc.c  | 2 +-
 drivers/video/rockchip/mpp/mpp_service.c | 6 +++---
 drivers/video/rockchip/mpp/mpp_vepu2.c   | 2 +-
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/video/rockchip/mpp/mpp_rkvenc.c b/drivers/video/rockchip/mpp/mpp_rkvenc.c
index 8f9d8cbcf7aa..5248060e005e 100644
--- a/drivers/video/rockchip/mpp/mpp_rkvenc.c
+++ b/drivers/video/rockchip/mpp/mpp_rkvenc.c
@@ -798,7 +798,7 @@ static int rkvenc_procfs_init(struct mpp_dev *mpp)
 	mpp_procfs_create_u32("session_buffers", 0644,
 			      enc->procfs, &mpp->session_max_buffers);
 	/* for show session info */
-	proc_create_single_data("session_info", 0444,
+	proc_create_single_data("sessions-info", 0444,
 				enc->procfs, rkvenc_show_session_info, mpp);
 
 	return 0;
diff --git a/drivers/video/rockchip/mpp/mpp_service.c b/drivers/video/rockchip/mpp/mpp_service.c
index 2d49e2efaf45..f0052f76a6c0 100644
--- a/drivers/video/rockchip/mpp/mpp_service.c
+++ b/drivers/video/rockchip/mpp/mpp_service.c
@@ -241,12 +241,12 @@ static int mpp_procfs_init(struct mpp_service *srv)
 	/* show version */
 	proc_create_single("version", 0444, srv->procfs, mpp_show_version);
 	/* for show session info */
-	proc_create_single_data("session_summary", 0444,
+	proc_create_single_data("sessions-summary", 0444,
 				srv->procfs, mpp_show_session_summary, srv);
 	/* show support dev cmd */
-	proc_create_single("support_cmd", 0444, srv->procfs, mpp_show_support_cmd);
+	proc_create_single("supports-cmd", 0444, srv->procfs, mpp_show_support_cmd);
 	/* show support devices */
-	proc_create_single_data("support_device", 0444,
+	proc_create_single_data("supports-device", 0444,
 				srv->procfs, mpp_show_support_device, srv);
 
 	return 0;
diff --git a/drivers/video/rockchip/mpp/mpp_vepu2.c b/drivers/video/rockchip/mpp/mpp_vepu2.c
index 2cbc72f27b9a..af896645c8cd 100644
--- a/drivers/video/rockchip/mpp/mpp_vepu2.c
+++ b/drivers/video/rockchip/mpp/mpp_vepu2.c
@@ -581,7 +581,7 @@ static int vepu_procfs_init(struct mpp_dev *mpp)
 	mpp_procfs_create_u32("session_buffers", 0644,
 			      enc->procfs, &mpp->session_max_buffers);
 	/* for show session info */
-	proc_create_single_data("session_info", 0444,
+	proc_create_single_data("sessions-info", 0444,
 				enc->procfs, vepu_show_session_info, mpp);
 
 	return 0;
-- 
2.35.3

